proxy.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import argparse
+from flask import Flask
+from flask import request
+import requests
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--whitelist', default='whitelist.txt',
+                        help='whitelist file path (default: \"whiltelist.txt\")')
+    parser.add_argument('--bind', default='0.0.0.0',
+                        help='binding address (default: 0.0.0.0)')
+    parser.add_argument('--port', type=int,
+                        help='binding port')
+    parser.add_argument('--forward', type=int, default=8080,
+                        help='port to forward (deafult: 8080)')
+
+    args = parser.parse_args()
+
+    app = Flask(__name__)
+    with open(args.whitelist) as f:
+        app.whitelist = set(line for line in f.readlines())
```

Comment:
It must be
```python
with open(args.whitelist) as f:
    app.whitelist = set(line.rstrip() for line in f)
```

Reply:
The current code doesn't remove the new line.

Reply:
Fixed it.